README.md

```diff
@@ -2,27 +2,844 @@
+   * Objects
+      * `start` - Object with the time the form is openned for editing
+         * `value` - Date time value in the format `yyyy-MM-dd HH:mm:ss`
+      * `end` - Time the form is completed
+         * `value` - Date time value in the format `yyyy-MM-dd HH:mm:ss`
+      * `today` - Day the form was completed
+         * `value` - Date value in the format `dd-MM-yyyy`
+      * `deviceid` - Unique identifier of the provider device
+      * `subscriberid` - Unique Subscriber ID based on Service Provider eg. `IMSI` will be returned for a `GSM Phone`
+      * `simserial` - SIM card Serial Number, if one is available
+      * `phonenumber` - Phone number for SIM 1 eg. the `MSISDN` for a `GSM Phone`
+      * `encounter_location` - Health Facility Zone where the encounter happened
+      * `look_up` - JSON Object describing any lookup data selected in a form lookup field. It basically has:
+         * `entity_id` - The unique entity name eg. `mother`
+         * `value` - The unique entity identifier eg. `9898-sd23D-f523a`
+
+> ### OpenMRS Keys
+> 
+> Each metadata field objects above have the following properties:
+>    * `key` 
+>    * `openmrs_entity_parent`
+>    * `openmrs_entity`
+>    * `openmrs_data_type` 
+>    * `openmrs_entity_id`
+
+3. Mandatory fields for each JSON Form i.e.
+   * `encounter_type` - This can be either of the [encounter types below](#encounter-types) or `blank` if the encounter type is not supported by OpenMRS
+   * `entity_id` - Unique identifier
+
+### Encounter Types
+
+Below are the most common encounter types:
+
+Encounter Type | Description
+-------------- | -----------
+Birth Registration | Child enrollment
+AEFI | Adverse Effect of vaccines, supplements or other services given by providers
+HIA2 Monthly Report | Monthly health reports written by providers
+Out of Catchment Service | Health service provision to a patient outside the patient's registered location eg. During `relative visits`, `temporary relocation`
+Death | A deceased child/patient is reported
+
+
+4. Validations & Constraints for input fields i.e.
+   * `v_regex` - This is used for validating input using [Regular Expression](https://en.wikipedia.org/wiki/Regular_expression) 
+   * `v_min` - This validation ensures input entered is not below minimum value stated
+   * `v_max` - Ensure that the input entered is not above the maximum value stated
+
+5. Different types of comparisons were added for the **Skip Logic**
+
+This enables a field to be shown only if the condition set in `'ex'` is `True`
+
+Skip Logic is defined using in `"relevance"` value
+
+#### Sample Skip Logic
+
+```
+"relevance": {
+          "step1:Place_Birth": {
+            "type": "string",
+            "ex": "equalTo(., \"Health facility\")"
+          }
+        }
+```
+
+The field above will only be shown if the (key) `Place_Birth` value, which is **Patient's Place of Birth(Location)**, is equals to `"Health Facility"`
```

Comment:
remove parens around "key"

Reply:
* add "the" after "which is"
* add space after "Birth" before open paren

Reply:
**Before:** The field above will only be shown if the (key) `Place_Birth` value, which is **Patient's Place of Birth(Location)**, is equals to `"Health Facility"`

**After:** The field above will only be shown if the key `Place_Birth` value, which is the **Patient's Place of Birth (Location)**, is equals to `"Health Facility"` 

After removing the parenthesis, the statement seems not right. Advise on this

Reply:
it does seem a little off, I'd rearrange as

> The field above will only be shown if the value for the key Place_Birth, which is the Patient's Place of Birth (Location), is equals to "Health Facility"

Reply:
👍